### DISCUSSION TITLE
Новини квітня, 2024

### DISCUSSION THREAD

**Discussion by ikalnytskyi** [shopokodu/community#58] [Теми для подкасту]
- [x] [Поттерінг перевинайшов sudo](https://mastodon.social/@pid_eins/112353324518585654)
- [x] [Leaving Rust gamedev after 3 years](https://loglog.games/blog/leaving-rust-gamedev/)
- [x] [Схема видобутку грошей від AWS](https://medium.com/@maciej.pocwierz/how-an-empty-s3-bucket-can-make-your-aws-bill-explode-934a383cb8b1)
- [x] https://github.com/microsoft/MS-DOS
- [x] https://world.hey.com/dhh/we-re-moving-continuous-integration-back-to-developer-machines-3ac6c611
- [x] https://fortune.com/europe/2024/04/23/spotify-earnings-q1-ceo-daniel-eklaying-off-1500-spotify-employees-negatively-affected-streaming-giants-operations/
- [x] [Why reading whitepapers takes your career to the next level](https://read.highgrowthengineer.com/p/why-reading-whitepapers-takes-your)
- [x] [Tomorrow Corporation Tech Demo](https://youtu.be/72y2EC5fkcE?si=27j-VatgI5kPc-xj)

**Comment by dolifer** [comment]
[Google layoffs hit Flutter team](https://news.ycombinator.com/item?id=40184763)

[Leaving Rust gamedev after 3 years](https://loglog.games/blog/leaving-rust-gamedev/)

- [Tomorrow Corporation Tech Demo](https://youtu.be/72y2EC5fkcE?si=27j-VatgI5kPc-xj)

[How an empty S3 bucket can make your AWS bill explode](https://medium.com/@maciej.pocwierz/how-an-empty-s3-bucket-can-make-your-aws-bill-explode-934a383cb8b1)
